test/image/mocks/heatmap-with-zero-category.json

```diff
@@ -0,0 +1,32 @@
+{
+  "data": [
```

Comment:
This mock currently renders as:

![image](https://user-images.githubusercontent.com/6675409/55185885-ad28e600-516b-11e9-8df2-7f8354de3aff.png)